_episodes/03-dplyr.md

```diff
@@ -66,84 +73,78 @@ To load the package type:
 
 The package **`tidyr`** addresses the common problem of wanting to reshape your data for plotting and use by different R functions. Sometimes we want data sets where we have one row per measurement. Sometimes we want a data frame where each measurement type has its own column, and rows are instead more aggregated groups - like plots or aquaria. Moving back and forth between these formats is nontrivial, and **`tidyr`** gives you tools for this and more sophisticated  data manipulation.
 
-To learn more about **`dplyr`** and **`tidyr`** after the workshop, you may want to check out this
-[handy data transformation with **`dplyr`** cheatsheet](https://github.com/rstudio/cheatsheets/raw/master/data-transformation.pdf) and this [one about **`tidyr`**](https://github.com/rstudio/cheatsheets/raw/master/data-import.pdf).
+To learn more about **`dplyr`** and **`tidyr`** after the workshop, you may want to check out this [handy data transformation with **`dplyr`** cheatsheet](https://github.com/rstudio/cheatsheets/raw/master/data-transformation.pdf) and this [one about **`tidyr`**](https://github.com/rstudio/cheatsheets/raw/master/data-import.pdf).
 
 We'll read in our data using the `read_csv()` function, from the tidyverse package **`readr`**, instead of `read.csv()`.
 
 
-
 ```r
-surveys <- read_csv("data/portal_data_joined.csv")
+plant_phys <- read_csv("data/Physiology_Environmental_Data.csv")
+```
+
+```
+## Warning: Missing column names filled in: 'X19' [19]
 ```
 
 ```
 ## Parsed with column specification:
 ## cols(
-##   record_id = col_integer(),
-##   month = col_integer(),
-##   day = col_integer(),
-##   year = col_integer(),
-##   plot_id = col_integer(),
-##   species_id = col_character(),
-##   sex = col_character(),
-##   hindfoot_length = col_integer(),
-##   weight = col_integer(),
-##   genus = col_character(),
-##   species = col_character(),
-##   taxa = col_character(),
-##   plot_type = col_character()
+##   Year = col_integer(),
+##   DOY = col_integer(),
+##   Species = col_character(),
+##   Fgroup = col_character(),
+##   Cond_night = col_double(),
+##   Trmmol_night = col_double(),
+##   Photo = col_double(),
+##   Cond_day = col_double(),
+##   Trmmol_day = col_double(),
+##   Percent_cond = col_double(),
+##   Percent_trmmol = col_double(),
+##   PD = col_double(),
+##   MD = col_double(),
+##   Soil_moisture = col_double(),
+##   TAIR_N = col_double(),
+##   VPD_N = col_double(),
+##   TAIR_D = col_double(),
+##   VPD_D = col_double(),
+##   X19 = col_character()
 ## )
 ```
 
 ```r
 ## inspect the data
-str(surveys)
+str(plant_phys)
 ```
 
+We also need to load the iris data as a tibble for this lesson
+
 
 ```r
-## preview the data
-View(surveys)
+data(iris)
```

Comment:
I added data/iris.csv.   Might it be better to use read_csv("data/iris.csv")  here instead of using a different data import method?